### ISSUE TITLE
VoiceOver for macOS: Make toggle button rendering consistent with VO iOS where both pressed and not pressed states are conveyed (Track Apple rdar://100389758)

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at#1139]
From #784 where

              >> Make VoiceOver's rendering of toggle buttons consistent with their treatment in iOS Safari, where VoiceOver communicates both states.

> This is a reasonable request that I will discuss with the Mac and iOS teams. Thanks.

For closure, the Apple-internal tracking numbers are rdar://100389758 (for Mac) and rdar://100389766 (for iOS). Those teams will discuss potentially converging on either the Mac or iOS behavior, or some variant in between. I am not able to predict a timeline for that discussion.

In the meantime, I believe the test expectations should be changed to match the existing VoiceOver behavior. Thanks.

_Originally posted by @cookiecrook in https://github.com/w3c/aria-at/issues/784#issuecomment-1257330397_
